app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,5 @@
+class FoodsController < ApplicationController
+  def index
+    @food = FoodService.all_food
```

Comment:
In order to properly make our query, we need to pass the user's query param from the controller to the service. We could do it like this:
`@food = FoodService.all_food(params[:q])`